.gitignore

```diff
@@ -1,2 +1,3 @@
-vendor
```

Comment:
We don't `vendor` our deps in this project, leave this in and remove the `vendor/` folder

Reply:
Sure, I will drop that.